plugins/obs-ffmpeg/obs-ffmpeg.c

```diff
@@ -59,11 +55,10 @@ extern struct obs_encoder_info hevc_vaapi_encoder_tex_info;
 #endif
 #endif
 
-#ifndef __APPLE__
+#ifdef ENABLE_FFMPEG_NVENC
```

Comment:
Trying to understand what currently happens if you tried to compile on macOS and what would happen in this PR if you set `ENABLE_FFMPEG_NVENC` to `ON` on macOS.

Reply:
I think right now this would just result in it checking if the nvenc encoder is available in FFmpeg (which it wouldn't be), all the OS-specific code is ifdef'd separately.